apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -120,11 +120,21 @@ type OpenTelemetryTargetAllocator struct {
 	// +optional
 	Enabled bool `json:"enabled,omitempty"`
 
+	// PrometheusCR defines the configuration for Prometheus custom resource retrieval.
```

Comment:
What CR is it exactly? could you please document the fully qualified name (group.kind)? (I am playing an ignorant here not knowing anything about p8s)

Reply:
It's referring to the `ServiceMonitor` and `PodMonitor` CRDs in the `monitoring.coreos.com/v1` API